lively.morphic/morph.js

```diff
@@ -2236,6 +2236,20 @@ export class Morph {
     evt.hand.dropMorphsOn(this);
   }
 
+  doNotAcceptDropsForThisAndSubmorphs() {
+    this.acceptsDrops = false;
+    this.submorphs.forEach((m) => {
```

Comment:
Interesting, I was expecting this to not only look at the direct children of the morph in question, but the entire submorph hierarchy, so all children reachable from that morph.
I can imagine a bunch of scenarios where I have to manually go through each layer of morphs manually and disable the drops, which kind of defeats the purpose of this "shortcut"? 
For instance, calling this on a list will still leave the items in that list unchanged (and continue to accept drops), since these are wrapped inside a list container.
Then again, it avoids unintended disabling of drops, since a smaller set of morphs is altered.
Thoughts?

Reply:
I am not sure if I understand you correctly, but I actually thought we traverse the whole submorph-hierarchy, since we have the recursive call in line 2242?

Reply:
Beats me! You are absolutely right. Please use the idiomatic way in lively which is: `this.withAllSubmorphsDo(m => {})`
However then we end up with the other side of the issues I mentioned. For instance opening up a drag and drop game inside a window would break the game.